docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -66,6 +66,40 @@ Result:
 
 -   [Map(key, value)](../../sql-reference/data-types/map.md) data type
 
+## mapFromArrays
+
+mapFromArrays merges an array of keys and an array of values into a Map.
+  
```

Comment:
I think we should mention @CurtizJ's example from  8 Nov 21. Something like:

```md
The function is a more convenient alternative to `CAST((keys, values), 'Map(key_type, value_type)')`. For example, instead of writing `CAST((['aa', 'bb'], [4, 5]), 'Map(String, UInt32)')`, you can write `mapFromArrays(['aa', 'bb'], [4, 5])`.
```